### DISCUSSION TITLE
License of assets is ambiguous

### DISCUSSION THREAD

**Discussion by YeldhamDev** [Bithack/principia#180] [General]
While the license of the source code is clear (BSD-3), the license of the assets used isn't (textures and stuff). Are they also under the same license?

**Comment by pajlada** [comment]
I would recommend working under the assumption that you cannot redistribute assets unless otherwise stated in the repo.

From my limited memory, the sounds should mostly be CC0 sounds from freesound.org, or made by us in Audacity
Models should all be made by us in Blender
Fonts are not made by us, but free to use according to their licenses (which should probably be linked to in the fonts directory)
I don't know where the textures are from, but given the places we fetched sounds I suspect is from a similar website containing CC0 high def textures.
